bin/drain_cleaner_run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+set +x
+
+# Deny illegal access option is supported only on Java 9 and higher
+JAVA_OPTS="${JAVA_OPTS} --illegal-access=deny"
+
+# Exit when we run out of heap memory
+JAVA_OPTS="${JAVA_OPTS} -XX:+ExitOnOutOfMemoryError"
+
+# Disable FIPS if needed
+if [ "$FIPS_MODE" = "disabled" ]; then
+    export JAVA_OPTS="${JAVA_OPTS} -Dcom.redhat.fips=false"
```

Comment:
Is this `com.redhat.fips` thing because we're using some Red Hat build of OpenJDK from UBI minimal which adds some FIPS awareness?

Reply:
Yes, correct. This is because we use the Red Hat OpenJDK from the UBI base image.